.github/workflows/test-python.yml

```diff
@@ -8,24 +8,43 @@ jobs:
 
     name: Run all unit tests in code base
     runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
 
     env:
       PYTHON_VERSION: 3.8
 
     steps:
     - uses: actions/checkout@v2
+    - name: Cache conda env
+      uses: actions/cache@v2
+      env:
+        # Increase this value to reset cache if environment_linux.yml has not changed
```

Comment:
@ayushbaid should this say "if environment_linux.yml has changed" or "has not changed"?

Reply:
If the file has changed, the hash value will change and there will be a cache miss. This comment is to invalidate the cache even when the file is exactly the same.